src/routing/mod.rs

```diff
@@ -312,17 +311,24 @@ impl Routing {
         dst: DstLocation,
         content: Bytes,
     ) -> Result<()> {
+        if let DstLocation::EndUser(EndUser::Client { socket_id, .. }) = dst {
```

Comment:
bit confused here.
with this block of code, do we still need the following `send_message_to_client` ?

Reply:
Yeah IDK, this was the _super simple_ `first-get-it-working` code, to expose one single `send_message` API.
It's because I didn't know exactly how the internals should be refactored.